src/common/resources/organization.js

```diff
@@ -28,4 +28,4 @@ export const fetchOrgsByParam = {
   fetch: false,
 };
 
-export const RESULT_COUNT_INCREMENT = 30;
+export const RESULT_COUNT_INCREMENT = 100;
```

Comment:
why 100?

Reply:
I see from requirements, need to discuss with Dennis, 100 can be expensive on some screens

Reply:
Ok)